tox.ini

```diff
@@ -19,7 +19,7 @@ deps =
     --requirement deps/docs.txt
 commands =
     pip install --editable .
-    sphinx-build -W --keep-going docs-src docs/docs {posargs}
+    sphinx-build -W --keep-going docs-src {env:HYPOFUZZ_DOCS_OUTPUT_DIR:docs/docs} {posargs}
```

Comment:
I think we'll also need to configure `passenv` for tox

Reply:
great call thanks, this probably saved me a lot of frustrated debugging